tests/nsis3/share/nsis/Include/WinCore.nsh

```diff
@@ -167,7 +167,7 @@ ShlObj.h
 !define CSIDL_COMMON_DESKTOPDIRECTORY 0x0019 ;All Users\Desktop
 !define CSIDL_APPDATA                 0x001a ;<user name>\Application Data
 !define CSIDL_PRINTHOOD               0x001b ;<user name>\PrintHood
-!define CSIDL_LOCAL_APPDATA           0x001c ;<user name>\Local Settings\Applicaiton Data (non roaming)
+!define CSIDL_LOCAL_APPDATA           0x001c ;<user name>\Local Settings\Application Data (non roaming)
```

Comment:
⚠ This might break compatibility. Revert?

Reply:
This is just a comment, isn't it? The constant is defined as 0x001c irrespective of the prose coming after?

If @twoln confirms this then I'd accept the PR as-is.

Reply:
You're probably correct. I had simply overlooked that `;` starts comments.